### DISCUSSION TITLE
Not able to start/scan "Freestyle Libre 3, Scan Error" (EU Version) (standalone mode)

### DISCUSSION THREAD

**Discussion by Li-nuz** [maheini/FreeStyle-Libre-3-patch#57] [Q&A]
I'm from Germany and a Libre 3 user for quite a while now. My usual setup was an old patched LibreLink app (Vers 3.3.0) with Juggluco implementation, the Juggluco app itself with accordingly configured ports to receive the data from the patched LibreLink app and then sending the received data from Juggluco to xDrip+ ...so three apps just to get the live raw data fom the sensor.

Therefore, I wanted to upgrade to a more modern approach, after reading Juggluco can finally start and use the libre 3 standalone without LibreLink.  ...nice!

**The problem:** I just can't get the Sensor to start/scan. The only thing I get when trying to start a sensor is **"Freestyle Libre 3, Scan Error. Try again."**

I tried the newest version Juggluco 7.1.9 and even tried an older version Juggluco 7.0.2.
I'm using a OnePlus 9 running Android 11. All permission are allowed, battery optimisation is disabled, location, bluetooth, nfc were turned on/off/on, restarts were performed etc. No success...

Is this a region problem? (eu sensor charge) ....or am I missing anything?

**Comment by maheini** [comment]
Hi @Li-nuz 
It should be working... that's weird. Which app does this message comes from? Is there any chance, a other app is receiving the NFC tag? Have you started the sensor with the Libre 3 app before (isn't necessary, but might cause other problems)? 

Please share your process of activating it and maybe a screenshot of the warning message. Do you still got Libre 3 on your phone?

**Reply by Li-nuz** [reply]
Thanks for the fast reply!

My process was:

- Uninstalled LibreLink and old Juggluco versions
- Downloaded and installed latest Juggluco app (Vers. 7.1.9)
- Placed a brand new Freestyle Libre 3 Sensor (not activated yet ofc)
- Changed the settings according to the instructions mentioned in the main [README.md](https://github.com/maheini/FreeStyle-Libre-3-patch#readme)
- Gave the location permissions (had to do it manually in the App Permission Section in the Android Settings as the pop-up didn't actually toogle the permission after pressing the button)
- Disabled battery optimization in the app settings as well
- Then scanned the sensor with NFC

= wild vibrations for about a second and then the screen changed to all-yellow and with text saying "Freestyle Libre 3, Scan Error. Try again."

Is the yellow screen from Juggluco app or is this maybe from the xDrip app that gets triggered by the nfc scan?

Next Steps:
1. I'll uninstall xDrip and try again
2. I'll start the sensor with librelink and try Juggluco to take over the sensor

I'll let you know how it wen't

**Reply by maheini** [reply]
In order to determine the app scanning the NFC sensor, close all apps and open Juggluco. Try scanning the sensor, whilst holding your hand steady and wait for the error to happen. If you got the problem, keep it open and trigger a app-switch. Now you should be able to determine if there is any new app open, or at least which app is displaying the error.

Your 2nd approach might work if the sensor start won't succeed in Juggluco.

Btw.: Which version of the App are you using? There's a version without Libre 3 support as well, please make sure you are using the one designed for libre 3. If you aren't sure, [try this download link of version 7.1.9 ARM](https://drive.google.com/open?id=1SMZYYfto1zvOSXPLhG4XO7UTOdgkJ1wk).

**Reply by Li-nuz** [reply]
As mentioned, I was using the latest version Juggluco 7.1.9. 

- I tried to be as steady as possible with the NFC. My OnePlus 9 actually has quite a strong NFC module that works flawlessly usually.
- I tried your method, the yello screen with the error promt is definitely coming from the Juggluco app. I even uninstalled xDrip+ but no difference.

= It looks like this one but with the text mentioned earlier:
![Screenshot 2023-11-13 235428](https://github.com/maheini/FreeStyle-Libre-3-patch/assets/54778964/b2ff8e38-81fc-46d0-9344-a842d1cb2c5f)

_______________________________________________________________________________

- I tried to start the sensor with the LibreLink app first and then switch over
1. waited 60 minutes for sensor warmup
2. closed the LibreLink app 
3. stopped the LibreLink app in the system settings
4. scanned the sensor in Juggluco
= same Problem

_______________________________________________________________________________

- Then I tried the other process explained in this github with a connected LibreView Account.
- But now I get a different error screen regarding the LibreView Id but i checked several times if I entered the right information:
![Screenshot 2023-11-14 001145](https://github.com/maheini/FreeStyle-Libre-3-patch/assets/54778964/74ee6423-4564-4537-9ad1-a103d3b963b0)

**Reply by durgok** [reply]
@Li-nuz Hello I have the same error. What did you do to fix it? Please help.

**Reply by Li-nuz** [reply]
@durgok I unfortunately couldn't fix the issue. Therefore, i temporarily went back to the modded Librelink + Juggluco + xDrip. Shortly after I switched the whole system to the Dexcom G7, which works directly with xDrip and also was way more accurate for me personally.

**Reply by durgok** [reply]
In my case, the issue was resolved once I upgraded to the latest version of Juggluco. I was not thinking that would be the cause but then once I fixed it, it was solved

**Comment by geecko86** [comment]
Having the exact same issue on Android 13, all permissions granted - though location was never requested.

**Reply by maheini** [reply]
Thank you for your report - I can't seem to find any problems for me, therefore it might be best to contact j-kaltes by submitting an issue on the Juggluco repository: [Submit Issue](https://github.com/j-kaltes/Juggluco/issues).

**Comment by Swisskitty** [comment]
Ich habe neu ein Xiaomi Redmi Note 13 plus pro, bei mir funktioniert das Scannen des Sensors Libre 3 nicht, da das NFC nicht kompatibel ist mit dem Sensor. Ich habe mich die letzten Tage mit Google schlau gemacht, und das ist ein bekanntes Problem für dieses Handy und scheinbar nicht nur für dieses. Was ich gelesen habe funktioniert es weder mit der Libre App, kann ich bestätigen, noch mit der gepatchten oder diesem Juggluco. Ich habe jetzt gelesen das wen man dieses Juggluco benutzt, man den Sensor mit einem anderen Handy aktivieren kann, und es dann wenn man sich auf dem normalen Handy in der App anmeldet es dann dort auch funktioniert. Ich möchte nicht die ganze Zeit 2 Geräte mitschleppen, Aber weiss jemand ob das funktioniert, ohne das man den Sensor wieder scannen muss. Bei der Libre App könnte man ja den Sensor auf einem anderen Handy starten, aber damit er dann auf dem normalen Handy funktioniert muss man ihn scannen, was ja leider nicht funktioniert. Mir wäre die offizielle Libre App am liebsten, da am einfachsten, bis man dieses Juggluco, XDriv und was es sonst noch braucht, hat, und zum funktionieren bringen sollte man es dann auch noch, ist furchtbar kompliziert und bin da echt überfordert.
